src/Sylius/Bundle/CoreBundle/Fixture/Factory/PromotionExampleFactory.php

```diff
@@ -45,11 +50,13 @@ class PromotionExampleFactory extends AbstractExampleFactory implements ExampleF
 
     public function __construct(
         FactoryInterface $promotionFactory,
+        FactoryInterface $couponFactory,
```

Comment:
Probably, we should inject it as the last nullable parameter, to prevent from BC break. Check out [this approach](https://github.com/Sylius/Sylius/blob/b157a18a06fae0a6eafba73b2ed9489a0fc3df21/src/Sylius/Bundle/CoreBundle/Fixture/Factory/AdminUserExampleFactory.php#L48).